www/src/components/main-content.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const MainContent = ({ children, ...props }) => {
```

Comment:
Prefer

```
export default function MainContent(…)
```

We've been inconsistent in the past, but going forward we should give components names: https://twitter.com/dan_abramov/status/1255229440860262400

Reply:
In addition, could you write a comment that describes what this component is used for? e.g.:

```
/**
 * <main> content wrapper that should be included in every page on the site.
 */
```